presto-thrift-connector/src/main/java/com/facebook/presto/connector/thrift/ThriftConnectorConfig.java

```diff
@@ -69,4 +70,16 @@ public ThriftConnectorConfig setLookupRequestsConcurrency(int lookupRequestsConc
         this.lookupRequestsConcurrency = lookupRequestsConcurrency;
         return this;
     }
+
+    public boolean shouldSetIdentityThriftHeaders()
```

Comment:
I think this might be the reason of test failures (@Config does not have getter), because the framework might rely on the getter to start with `is` for `boolean`.

How about we rename this config to `useIdentityThriftHeaders` (`presto-thrift.use-identity-thrift-headers`), and then getter becomes `isUseIdentityThriftHeaders` and setter becomes `setUseIdentityThriftHeader`?